help-service/src/main/resources/application-test.yml

```
@@ -0,0 +1,27 @@
+spring:
+  datasource:
+    driver-class-name: org.h2.Driver
+    url: jdbc:h2:mem:~/timebank
+    username: admin
+    password:
+  kafka:
+    bootstrap-servers: localhost:9092
+    listener:
+      concurrency: 4 # 🔧 반드시 토픽 partition 수보다 작지 않게
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)